examples/proof-of-concept/azure-iot/iot-hub-connector/pom.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  Copyright 2016-2020 ForgeRock AS. All Rights Reserved
```

Comment:
Is this the correct licence?

Reply:
The files are all created/added by a Maven archetype and this is the license that they are created with. I agree that it's not very useful, but I can also not change them.